src/Impostor.Server/Recorder/PacketRecorder.cs

```diff
@@ -20,10 +21,13 @@ namespace Impostor.Server.Recorder
     /// </summary>
     internal class PacketRecorder : BackgroundService
     {
+        private const int ProtocolVersion = 0;
```

Comment:
Would prefer if you started on `1`.

Reply:
Also make it an uint, would actually prefer it to be an enum so we can name versions according to their changes. UnrealEngine also does this for their internal versions, I think it's useful.

Reply:
Sounds interesting, will do